extensions/github-review-requests/src/actionablePullRequests.tsx

```diff
@@ -0,0 +1,50 @@
+import { Color, MenuBarExtra, open } from "@raycast/api";
+import usePulls from "./hooks/usePulls";
+import PullRequestItem from "./components/PullRequestItem";
+import { PullRequestShort } from "./types";
+
+const actionablePullRequests = () => {
+  const { isLoading, login, updatedPulls, recentlyVisitedPulls, visitPull } = usePulls();
+
+  const title = getTitle(updatedPulls);
+  const icon = getIcon(updatedPulls);
+
+  const showSeparator = updatedPulls.length > 0 && recentlyVisitedPulls.length > 0;
+
+  return (
+    <MenuBarExtra isLoading={isLoading} icon={icon} title={title}>
```

Comment:
if both `updatedPulls` and `recentlyVisitedPulls` are empty, then I get an `All good` item in the menu bar that (seemingly) doesn't react to being clicked. This feels a bit…confusing?
What I would suggest is either:
- always add a `No updated pulls` or similar item in this case, making sure that the menu bar extra has a menu
- or make the extension return `null` instead of a `MenuBarExtra`. This will result in the nothing being placed in the menu bar in this case, which…sounds appropriate? 🤔

Reply:
The first option sound better in this case. This would give an impression that something happens, and once visited a PR from the menu bar, there will always be a `Recently Visited` submenu item. Thank you for the suggestion!